managed/services/telemetry/config.default.yml

```diff
@@ -951,12 +948,19 @@ telemetry:
       - metric_name: "postgresql_db_count"
         value: 1
 
-  - id: PMMServerHAEnabled
```

Comment:
What is the reason to remove this metric?

Reply:
Oh, I didn't realize it was renamed to `PMM_TEST_HA_ENABLE`, so I put it back along with the new name.